*** Reading local file: /opt/airflow/logs/loganalyzer/analyze_tsla_data/2021-07-25T20:56:27.502169+00:00/1.log
[2021-07-26 20:56:34,945] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: loganalyzer.analyze_tsla_data 2021-07-25T20:56:27.502169+00:00 [queued]>
[2021-07-26 20:56:34,960] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: loganalyzer.analyze_tsla_data 2021-07-25T20:56:27.502169+00:00 [queued]>
[2021-07-26 20:56:34,961] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-26 20:56:34,961] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2021-07-26 20:56:34,962] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-26 20:56:34,980] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): analyze_tsla_data> on 2021-07-25T20:56:27.502169+00:00
[2021-07-26 20:56:34,984] {standard_task_runner.py:52} INFO - Started process 15523 to run task
[2021-07-26 20:56:34,988] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'loganalyzer', 'analyze_tsla_data', '2021-07-25T20:56:27.502169+00:00', '--job-id', '80', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/mini-project/LogAnalyzer.py', '--cfg-path', '/tmp/tmpblyoseo8', '--error-file', '/tmp/tmp214v99j_']
[2021-07-26 20:56:34,989] {standard_task_runner.py:77} INFO - Job 80: Subtask analyze_tsla_data
[2021-07-26 20:56:35,030] {logging_mixin.py:104} INFO - Running <TaskInstance: loganalyzer.analyze_tsla_data 2021-07-25T20:56:27.502169+00:00 [running]> on host 51316632f516
[2021-07-26 20:56:35,078] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=loganalyzer
AIRFLOW_CTX_TASK_ID=analyze_tsla_data
AIRFLOW_CTX_EXECUTION_DATE=2021-07-25T20:56:27.502169+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-25T20:56:27.502169+00:00
[2021-07-26 20:56:35,087] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-21T23:22:13.962972+00:00/1.log
[2021-07-26 20:56:35,090] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,094] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-22T23:05:11.419269+00:00/1.log
[2021-07-26 20:56:35,097] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,101] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-22T23:22:29.705612+00:00/1.log
[2021-07-26 20:56:35,104] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,108] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:06:06.371163+00:00/1.log
[2021-07-26 20:56:35,111] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,115] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:11:27.799660+00:00/1.log
[2021-07-26 20:56:35,117] {logging_mixin.py:104} INFO - Total number of errors: 1
[2021-07-26 20:56:35,118] {logging_mixin.py:104} INFO - Here are all the errors:
[2021-07-26 20:56:35,118] {logging_mixin.py:104} INFO - [2021-07-23 00:11:46,746] {taskinstance.py:1501} ERROR - Task failed with exception
[2021-07-26 20:56:35,118] {logging_mixin.py:104} INFO - 
[2021-07-26 20:56:35,123] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:15:19.156922+00:00/1.log
[2021-07-26 20:56:35,125] {logging_mixin.py:104} INFO - Total number of errors: 1
[2021-07-26 20:56:35,125] {logging_mixin.py:104} INFO - Here are all the errors:
[2021-07-26 20:56:35,126] {logging_mixin.py:104} INFO - [2021-07-23 00:15:31,589] {taskinstance.py:1501} ERROR - Task failed with exception
[2021-07-26 20:56:35,126] {logging_mixin.py:104} INFO - 
[2021-07-26 20:56:35,130] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:18:30.795903+00:00/1.log
[2021-07-26 20:56:35,133] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,137] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:26:06.269737+00:00/1.log
[2021-07-26 20:56:35,140] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,144] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-23T00:29:27.463255+00:00/1.log
[2021-07-26 20:56:35,147] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,151] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-24T00:06:06.371163+00:00/1.log
[2021-07-26 20:56:35,154] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,158] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-25T00:06:06.371163+00:00/1.log
[2021-07-26 20:56:35,161] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,166] {logging_mixin.py:104} INFO - Analysis for log file: /opt/***/logs/marketvol/download_tsla_data/2021-07-26T18:00:39.307851+00:00/1.log
[2021-07-26 20:56:35,169] {logging_mixin.py:104} INFO - No errors"
[2021-07-26 20:56:35,171] {python.py:151} INFO - Done. Returned value was: None
[2021-07-26 20:56:35,186] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=loganalyzer, task_id=analyze_tsla_data, execution_date=20210725T205627, start_date=20210726T205634, end_date=20210726T205635
[2021-07-26 20:56:35,215] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-26 20:56:35,243] {local_task_job.py:149} INFO - Task exited with return code 0
